### PR TITLE
Installations lesson: Provide descriptive link text

### PR DESCRIPTION
**Pull request by matthewcho399**
## Because

The purpose of this PR is to make links more descriptive and accessible, following the layout style guide. The changes were made only to Foundations/Prerequisites/Installations.

## This PR

- Simply just replaced each of the original text links with the updated text links within the TOP Foundations: Provide accessible, descriptive links Google Spreadsheet. 

## Issue

Related to #27681 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installations.md [hunk 1]
```diff
@@ -206,14 +206,14 @@ Here are some tips to help you get started in a virtual environment:
...
+  - Do look out for the ![Icon with a green turtle and a V](https://cdn.statically.io/gh/TheOdinProject/curriculum/96d534641514fe4d62aabe2919fac3c52cb286e7/foundations/installations/installations/imgs/17_turtle.png){: .inline-img} icon in the bottom right of your VM window. This signifies that something called Hyper-V is turned on. A thread on the VirtualBox forums describes [how to turn off Hyper-V completely](https://forums.virtualbox.org/viewtopic.php?f=25&t=99390). You want to have the icon of a chip with a V icon instead. If you don’t see either of these icons in the bottom right, you have to exit full screen mode to see them.
```
Comment (MaoShizhong) [inline foundations/installations/installations.md:216]:
```suggestion
  - Do look out for the ![Icon with a green turtle and a V](https://cdn.statically.io/gh/TheOdinProject/curriculum/96d534641514fe4d62aabe2919fac3c52cb286e7/foundations/installations/installations/imgs/17_turtle.png){: .inline-img} icon in the bottom right of your VM window. This signifies that something called Hyper-V is turned on. A thread on the VirtualBox forums describes [how to turn off Hyper-V completely](https://forums.virtualbox.org/viewtopic.php?f=25&t=99390). You want to have the icon of a chip with a V ![Icon of a chip with a V](https://cdn.statically.io/gh/TheOdinProject/curriculum/96d534641514fe4d62aabe2919fac3c52cb286e7/foundations/installations/installations/imgs/18_vboxV.png){: .inline-img} icon instead. If you don’t see either of these icons in the bottom right, you have to exit full screen mode to see them.
```
This is my error for leaving out - could you add this icon link in as above?

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thank you very much for doing this @matthewcho399!
Just one change needed as below. Nothing you did wrong, just something I realised I missed out of a link change (accidentally got rid of an icon we really do want present!)

If you can make the requested change and push that commit to your branch, it'll show up here and I can double check then merge if all good :+1:

File: foundations/installations/installations.md (hunk 1 above)
Comment (matthewcho399) [reply foundations/installations/installations.md]: Done!

**Review by MaoShizhong — APPROVED** [review]
Thank you very much for making these changes, and congrats on your first contribution here! :rocket:

**Pull request merged**